### DISCUSSION TITLE
hrp 的研发时间表是怎样的呢，目前计划中有哪些发布节点？

### DISCUSSION THREAD

**Discussion by zhuifengshen** [httprunner/httprunner#1185] [2 - 答疑（Q&A）]
hrp 的研发时间表是怎样的呢，目前计划中有哪些发布节点，功能开发优先级如何考量？

**Comment by debugtalk** [comment]
@zhuifengshen HttpRunner 从 v4.0 开始采用 GitHub 的里程碑（milestone）功能做版本规划，每个版本会设定为一个里程碑，例如 v4.1、v4.2。然后我们会把所有的 issue（包括新功能、优化、bugfix 等）规划到具体版本中，并在 GitHub 的 Projects 中进行管理。

详见：https://github.com/orgs/httprunner/projects/1
